### PR TITLE
FIX: remove all code related to old_deployment

### PR DESCRIPTION
**Pull request by satrio-hw**
xref #230

I've got fail test on flake8 when run pre-commit on my local, but no error. please tell me if I need to fix something related to this PR
cc: @marco-c

### CODE REVIEW

**Comment by marco-c** [comment]
There are a few test failures to fix, and some flake8 errors. You can see the logs at https://travis-ci.org/github/mozilla/mozci/builds/730913737.

**Comment by satrio-hw** [comment]
> There are a few test failures to fix, and some flake8 errors. You can see the logs at https://travis-ci.org/github/mozilla/mozci/builds/730913737.

I need a little help here.. so I fixed test failure and recheck flake8 using command `flake8`,
also to make sure I also run `flake8` to changed files
```
[in] flake8 mozci/util/taskcluster.py
[out] mozci/util/taskcluster.py:8:1: F401 'requests' imported but unused
[in] flake8 tests/test_push.py
[in] flake8 tests/test_task.py
```
I already installed `pre-commit` in my local and after test files fixed, it show no more error, but after pushing to repo I still got fail in flake8.. something that I miss by any chance?

**Comment by satrio-hw** [comment]
I think I got it now, where I was wrong..

File: tests/test_task.py [hunk 1]
```diff
@@ -33,13 +33,6 @@ def test_missing_artifacts(responses, create_task):
         status=404,
```
Comment (marco-c) [inline tests/test_task.py:33]: Could you remove the `# First we'll check the new deployment.` comment, since it no longer applies?

**Review by marco-c — APPROVED** [review]
Thanks!

**Pull request merged**